src/view/views/SimulatorView.css

```diff
@@ -36,9 +36,16 @@
 /* Left Panel */
 .simulator-view--panel:nth-child(1) {
   flex: 2;
+  margin-top: var(--toolbar-height);
 }
 
 /* Right Panel */
 .simulator-view--panel:nth-child(2) {
   flex: 2;
 }
+
+@media only screen and (max-width: 1500px) {
```

Comment:
@JamesWP is there a way to use a `var` here for `max-width` value so that we can keep the 2 `@media` queries in sync?

Reply:
> This looks great, Is there anything you want to do before we land this to master

@JamesWP I would've liked to couple the 2 `@media` values for `max-width` somehow, but I'm not too fussed about it. Good to land on my side.